ocaml/idl/datamodel.ml

```diff
@@ -2389,6 +2389,19 @@ let vm_call_plugin = call
 	~allowed_roles:_R_VM_OP
 	()
 
+let vm_import_from = call
+	~name:"import_from"
+	~in_product_since:rel_dundee
+	~doc:"Import an XVA from a URI"
+	~params:[String, "url", "The URL of the XVA file";
```

Comment:
Thanks. Yes, import seems reasonable to me. I'll correct the url/uri disparity too, I had changed my mind over which one to call it...